build/depends.py

```diff
@@ -703,6 +703,11 @@ def configure(self, build, conf):
             raise Exception(
                 "Could not find qtkeychain.")
 
+class LAME(Dependence):
+    def configure(self, build, conf):
+        if not conf.CheckLib(['libmp3lame', 'libmp3lame-static']):
+            raise Exception("Could not find libmp3lame.")
```

Comment:
Do we want to make this optional instead of required?

Reply:
I think we probably always want an MP3 encoder since the recording feature is non-optional and the broadcast feature is always enabled in our releases, so I don't see a reason to make it optional. Plus it would take more work to make it optional because the existing code that handled lame not being dynamically loadable was super hacky (just throws up modal error message boxes instead of properly graying out the MP3 option in the preferences).